tests/integration/targets/btrfs_subvolume/aliases

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Ansible Projec
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+azp/posix/vm
```

Comment:
The tests should also be assigned to one of the standard CI groups so they also run on RHEL (at least the RHEL versions that still support Btrfs, I think RHEL 7 still does):
```suggestion
azp/posix/3
azp/posix/vm
```

Reply:
You're correct, only RHEL 7 supports btrfs but 8 and 9 do not.

I'm actually not clear on the difference between posix 1, 2 and 3 or where in the configuration I can differentiate between tests running in a container and a full vm. Per earlier discussions on matrix/irc, these tests will only work in a vm.

Reply:
1, 2 and 3 are only there to balance tests so that running all of them is faster (since groups can run in parallel). The tests can also run in a container, but only in privileged containers, which aren't used in CI. (Since you already put `needs/privileged` in there, ansible-test will skip it on all unprivileged containers.)

Reply:
Thanks for the explanation, I'll add the above suggestion with the next set of changes.